backend/src/test/java/com/redhat/cloud/notifications/routers/EventServiceTest.java

```diff
@@ -479,6 +479,26 @@ void testInvalidLimit() {
                 .contentType(JSON);
     }
 
+    @Test
+    void shouldNotBeAllowedToGetEventLogs() {
+        Header identityHeader = mockRbac(DEFAULT_ACCOUNT_ID, "noaccess", NO_ACCESS);
+        given()
+                .header(identityHeader)
+                .when().get(PATH)
+                .then()
+                .statusCode(403);
+    }
+
+    @Test
+    void shouldBeAllowedToGetEventLogs() {
+        Header noAccessIdentityHeader = mockRbac("tenant", "user", FULL_ACCESS);
+        given()
+                .header(noAccessIdentityHeader)
+                .when().get(PATH)
+                .then()
+                .statusCode(200);
+    }
```

Comment:
The `FULL_ACCESS` case is already tested multiple times in `EventServiceTest`.

Reply:
so what else do you want to be tested then?

Reply:
See my comment above:
> What I meant with my previous comment is that we should make sure that someone with the RBAC_READ_NOTIFICATIONS role can no longer access the event log API.

Reply:
`FULL_ACCESS` is already tested by the existing code. We don't need an additional test for that.

Reply:
deleted